### PR TITLE
Kate Hertweck answers to Chapter 2

### PR DESCRIPTION
**Pull request by k8hertweck**
I started off strong and then began to lose steam....

### CODE REVIEW

File: _exercises/02-03-looking-into-yourself.md [hunk 1]
```diff
@@ -29,3 +29,35 @@ What do we get rid of?
...
+    Sometimes things won't get done on time, or at a level of clarity/accomplishment that I would personally offer.
+
+4.  What specific behavior(s) can you practice to change this attitude?
+    
+    Delegation of tasks, 
+
+5.  When, specifically, will I commit myself to practicing this behavior or these behaviors?
+    
+    At meetings (I will not volunteer for a task).
+
```
Comment (ChristinaLK) [inline _exercises/02-03-looking-into-yourself.md]: I almost wrote this one down too!  Delegating = one half handing something over, one half not even signing yourself up in the first place.  ;)

**Comment by raynamharris** [comment]
Thanks Kate for the comment and for powering through. I wanted to quit and say all these exercises are ridiculous, but I read your answers and they inspired to to put my thoughts to the keys.

File: _exercises/02-02-finding-the-stories-that-guide-you.md [hunk 2]
```diff
@@ -28,3 +28,27 @@ This exercise is one way to start to learn and tell your story.
...
..., and have spent time teaching semester-long classes (in bioinformatics) to learn about particular challenges my students face. On one occassion, I talked with a handful of students before class started about their educational backgrounds. Their challenges were staggering: one student was homeschooled as an adolescent because, as an immigrant from the UK, her accent was mocked by her peers. Another student struggled with a Latin heritage (and name), but criticism from her peers because she was "too white." Another had moved to our region from out of state and couldn't quite grasp the unspoken nuances of how the culture worked. I had a student (graduating senior!) who was verbally appreciative that an instructor would ask and remember his preferred name. Some of my students have no idea of a career path and are desperate for marketable skills. All of these issues are related to how they perceive my class materials and are motivated to learn them (or even if they think it's really possible). I see my role as an educator and leader as recognizing the diversity of students with which we interact, and finding ways to best serve them (or even get them into our classrooms to begin with!).
```
Comment (gvwilson) [inline _exercises/02-02-finding-the-stories-that-guide-you.md]: Amen.

File: _exercises/02-02-finding-the-stories-that-guide-you.md [hunk 3]
```diff
@@ -28,3 +28,27 @@ This exercise is one way to start to learn and tell your story.
...
+
+4.  What does this story mean to you?
+    Think about why you chose this story.
+
+    I am frustrated by invisible barriers to student success and classism associated with education (especially higher-ed). I am frustrated by instructors who are only capable of teaching students that are like them. I am frustrated by the lip service paid to serving underrepresented groups without any effort to understanding what it is they need and how to accomplish these tasks, and by these efforts constantly taking a back seat to self-service. I want to do better.
```
Comment (gvwilson) [inline _exercises/02-02-finding-the-stories-that-guide-you.md]: How often/how much have you been obstructed by these barriers yourself?

File: _exercises/02-05-a-building-your-core-group.md [hunk 4]
```diff
@@ -39,3 +39,33 @@ title: "Building Your Core Group"
...
+    *   Yes
+
+6.  I tend to think first and plan carefully.
+    *   Yes
+
+7.  I tend to want to act first and plan later.
+    *   No
+
+8.  I tend to weigh decisions carefully and act slowly.
+    *   Try not to? I don't know if I can assess this for myself, especially since I'm often time-constrained.
```
Comment (gvwilson) [inline _exercises/02-05-a-building-your-core-group.md]: Who could you turn to?

File: _exercises/02-05-a-building-your-core-group.md [hunk 5]
```diff
@@ -39,3 +39,33 @@ title: "Building Your Core Group"
...
+
+9.  I tend to be intuitive.
+    *   Yes (but I don't always act on intuition).
+
+10. When I volunteer, I like to be a fly on the wall for awhile before speaking up or taking a more active role.
```
Comment (gvwilson) [inline _exercises/02-05-a-building-your-core-group.md]: How do you know when it's time to speak out?

File: _exercises/02-05-b-building-your-core-group.md [hunk 6]
```diff
@@ -23,3 +23,29 @@ title: "Building Your Core Group"
 
 6.  Who has these?
     [____________________]
+    
+## Kate Hertweck
+
+1.  Knowing your personality, what other kinds of personalities would be most useful in the core group?
+    
+    Creativity, diverse perspectives (other cultures, educational backgrounds)
```
Comment (gvwilson) [inline _exercises/02-05-b-building-your-core-group.md]: Do you think you're not creative???

File: _exercises/02-05-b-building-your-core-group.md [hunk 7]
```diff
@@ -23,3 +23,29 @@ title: "Building Your Core Group"
...
+    
+    Detail-oriented knowledge about areas I lack (finance, legal, etc)
+
+4.  Who has this knowledge or these skills?
+    
+    Hopefully other people elected to the Steering Committee?
+
+5.  What other skills or areas of knowledge are needed in the core group?
+    
+    Leadership and communication (especially regarding subcommittees)
```
Comment (gvwilson) [inline _exercises/02-05-b-building-your-core-group.md]: Do you need people who can raise their voices?

File: _exercises/02-05-c-building-your-core-group.md [hunk 8]
```diff
@@ -39,3 +39,47 @@ title: "Building Your Core Group"
...
+1.  What are the organization's essential principles?
+    (What are the deal-breaker agreements you require for any member of the core group?)
+    
+    Commitment to open availability of materials, Code of Conduct, model of teaching (and appropriate lessons)
+
+2.  What skills does the organization need?
+    *   Accounting (?)
+    *   Legal help (?)
+    *   Community support
+    *   Special knowledge of higher-level sysadmin/git/etc than we teach
```
Comment (gvwilson) [inline _exercises/02-05-c-building-your-core-group.md]: Communications

File: _exercises/02-03-looking-into-yourself.md (hunk 1 above)
Comment (k8hertweck) [inline _exercises/02-03-looking-into-yourself.md]: It's a tricky skill...a colleague reminded me awhile back to be careful not to train my students to fall into the same patterns I have...agreeing to do something and then getting overwhelmed. I try to be responsible in delegation :)

Comment (ChristinaLK) [inline _exercises/02-03-looking-into-yourself.md]: That's a good point - people are watching you.  I tend to think that no one is following my example, but there probably are some people!

File: _exercises/02-02-finding-the-stories-that-guide-you.md (hunk 3 above)
Comment (apawlik) [inline _exercises/02-02-finding-the-stories-that-guide-you.md]: @k8hertweck This is so good to read that! The lack of acknowledgement and understanding that there are many things (often very subtle and "invisible") that affect individual's educational opportunities and their learning performance is appallingly low. I myself only understood that when I went through the SWC/DC Instructor Training. Or actually when I started training new Instructors (and fully engaged with the material).

**Pull request merged**